### PR TITLE
Updated config.yml according to plots2 file

### LINKED ISSUE #1311: Update config.yml file for IS
**Please note we are preparing to participate in Google Code-in, and have reserved this issue for GCI participants - but we'd love to have your help with another one! Please check out https://code.publiclab.org to see more.**

Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update**

update the config.yml file present in .github of the home directory to follow the changes as per config.yml present in plots2 repo
https://github.com/publiclab/plots2/blob/master/.github/config.yml

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by sharmamansi**
Fixes #1311 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm run test-all`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below
* [x] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

### CODE REVIEW

**Comment by sharmamansi** [comment]
@publiclab/reviewers  Please review this.

File: .github/config.yml [hunk 1]
```diff
@@ -4,22 +4,27 @@
...
+ If you have screenshots to share demonstrating the issue, that's really helpful! 📸 You can [make a gif](https://www.cockos.com/licecap/) too!
+ Don't forget to join our [Gitter channel](https://gitter.im/publiclab/publiclab) for some brainstorming discussions. 
 # Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
 
 # Comment to be posted to on PRs from first time contributors in your repository
 newPRWelcomeComment: |
- Thanks for opening this pull request!
- There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
+ Thanks for opening this pull request! This space is [protected by our Code of Conduct](https://publiclab.org/conduct).
+ `Dangerbot` will test out your code and reply in a bit with some pointers and requests.There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
```
Comment (blurry-x-face) [inline .github/config.yml]: I don't think we use danger bot, so probably you can remove this, instea add this
`There may be some errors, **but don't worry!** We're here to help! 👍🎉😄`

**Comment by sharmamansi** [comment]
Sure will do it.

File: .github/config.yml [hunk 2]
```diff
@@ -4,22 +4,27 @@
 
 # Comment to be posted to on first time issues
 newIssueWelcomeComment: |
-  Thanks for opening your first issue here! Please follow the issue template to help us help you 👍🎉😄
-  If you have screenshots to share demonstrating the issue, that's really helpful! 📸 You can [make a gif](https://www.cockos.com/licecap/) too!
+ Thanks for opening your first issue here! This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.
+ Please follow the issue template to help us help you 👍🎉😄
+ If you have screenshots to share demonstrating the issue, that's really helpful! 📸 You can [make a gif](https://www.cockos.com/licecap/) too!
+ Don't forget to join our [Gitter channel](https://gitter.im/publiclab/publiclab) or (https://gitter.im/publiclab/image-sequencer)for some brainstorming discussions.
```
Comment (blurry-x-face) [inline .github/config.yml]: Replace `(https://gitter.im/publiclab/image-sequencer)` with `[ImageSequencer Gitter Channel](https://gitter.im/publiclab/image-sequencer)`

Comment (sharmamansi) [reply .github/config.yml]: Done!!!

**Comment by sharmamansi** [comment]
@LeoDog896 Done!! Please review it again

**Comment by debck** [comment]
LGTM

**Review by HarshKhandeparkar — APPROVED** [review]

**Comment by HarshKhandeparkar** [comment]
Thank you.

**Pull request merged**